package.json

```diff
@@ -128,6 +128,7 @@
         "intl": "1.2.5",
         "jasminewd2": "2.2.0",
         "karma-viewport": "1.0.8",
+        "lodash-es": "4.17.21",
```

Comment:
Please also update subpackages package.json files to reflect this dependency

Reply:
done